src/SqlExecutionArea.h

```diff
@@ -32,7 +34,8 @@ class SqlExecutionArea : public QWidget
     SqliteTableModel* getModel() { return model; }
     QTextEdit* getResultView();
     SqlTextEdit* getEditor();
-
+	ExtendedTableWidget *getTableResult();
```

Comment:
Ah, sorry. These are the (retarded) whitespace settings I had to configure for work.

Should be fixed